AUTHORING_MANIFESTS.md

```diff
@@ -77,3 +86,135 @@ It is important to test your manifest before submission to ensure it meets the r
+
+- This issue appears to be common to MSI installers. 
+- When installing to `%LOCALAPPDATA%` for `user` scope, the software package's registry entries are added to `HKEY_LOCAL_MACHINE` instead of `HKEY_CURRENT_USER`.
+- This makes the `user` scope software package appear to be of `machine` scope instead.
+  - Related to [Winget incorrectly detects per-user MSI scope (`No applicable update found`)](https://github.com/microsoft/winget-cli/issues/3011)
+- So a user can only get a "user" scope when installing for the first time, unless one of the below Workarounds is in place. 
+- This means, for subsequent upgrades through `winget upgrade`, the user will end up getting the "machine" scope installation. 
+  - A dual (or side-by-side) installation of the package (both user and machine scope) will result. 
+  - And both of these may show up in `winget list` and other places which list installed packages, causing confusion. Especially when one of the packages refuses to upgrade. 
+- Workarounds: 
+  - Package maintainers can remove `user` scope from packages, 
+  - Technical users can still retain the user scope install if they want by passing the args to the installer on update. For example:
+
+` winget upgrade Package.Identifier --custom "MSIINSTALLPERUSER=1"`
+
+### Upgrade always available for one package.
+
+This issue is slightly different from scope swapping above, in that there's only one version of the package installed. It can be caused by a few different situations:
+
+- Two manifests have the same data. If both the variants use the same DisplayName, ProductCode, Publisher and other package matching related information, then an existing install of a stable package may be mapped to a higher available version of a pre-release version by WinGet, even if the PackageIdentifiers are different. The fix is to remove the pre-release manifest, as the mainline has priority.
+- Version schema changes significantly - or changes between string and semantic. If a letter is added to one version number, then it can cause this issue with every version of the package. The fix is to remove manifests with the previous version schema, and only offer those with the latest schema.
+- Installer not writing accurate version data to the Windows Registry - either every installer writes the same version number, do not write a version number, do not write a consistent or accurate version number or have a similar issue with the version data in the registry. The fix here is to add the Apps and features metadata described above.
+
+### For a package that always writes the same version to registry: 
+
+Our package manager could update the Windows Registry, modify the installer nodes, and set DisplayVersion for the installer. But if the user or application performed any activity (reinstall, repair, update, uninstall, etc) outside of the package manager, then it would overwrite, or not write, this data. 
```

Comment:
Winget can't update the registry or set DisplayVersion, except for portable applications.